app/components/preprint-form-upload.js

```diff
@@ -10,8 +11,31 @@ export const State = Ember.Object.extend(Ember.Freezable, {
 }).create().freeze();
 
 const component = CpPanelBodyComponent.extend(PreprintFormFieldMixin, {
```

Comment:
Low priority, but nice to have when you need it:

If you expect a component to be reusable, it's highly beneficial to provide a docstring describing how to use the component (eg what input variables does it expect, and what data types should they be). This docstring should be the authority on how to use the component and be updated when usage changes.

Here is one basic example, in the YUIDoc format we use.
https://github.com/abought/ember-osf/blob/4f2947e7a0c3622b8f1c21d89e321caadf919b4e/addon/components/comment-detail/component.js#L24-L24